test/parallel/test-dgram-send-empty-array.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const dgram = require('dgram');
+
+const client = dgram.createSocket('udp4');
+
+const timer = setTimeout(function() {
+  throw new Error('Timeout');
+}, common.platformTimeout(200));
```

Comment:
Yes. It allows fast-failures, otherwise you rely on the 1 minute timeout that each tests has.